### DISCUSSION TITLE
Beatmap not found error should display more than the beatmap ID

### DISCUSSION THREAD

**Discussion by TheFantasticLoki** [ppy/osu#15359] [Ideas]
When creating a playlist and a map is added that is not submitted, the error should show more than Beatmap not found: 0,0,0 It should show the name of the map at the very least as no way to identify the problem map. 
This is only compounded by the fact there is no easy way to filter not submitted maps like there was in stable by typing status=not

[Screenshot](https://cdn.discordapp.com/attachments/639463329709948928/903552943935070218/unknown.png)

**Comment by bdach** [comment]
Agree that the error message is subpar.

> This is only compounded by the fact there is no easy way to filter not submitted maps like there was in stable by typing status=not

This is false however. You can do e.g. `status>none` to filter out non submitted maps I'm pretty sure.
